run_tests.py

```diff
@@ -0,0 +1,238 @@
+
+import argparse
+from run_riscv_tests import Colors
+import subprocess
+import sys
+import yaml
+
+CONFIG_PATH = 'example.yml'
+
+# Commands to run for riscv-tests
+RISCV_TESTS_BASE = [
```

Comment:
This seems like a nasty hack, spawning a sub-process Python rather than just making a new thread or something.

This indicates to me that we should turn `run_riscv_tests.py` into an object e.g. `RISCVTestRunner` that has parameters for all the flags. The standalone test script can then just parse args and build the object, and in this file we could just import it.